client/i18n/locales/english/intro.json

```diff
@@ -2037,7 +2037,10 @@
         ]
       },
       "uglc": { "title": "199", "intro": [] },
-      "vonu": { "title": "200", "intro": [] },
+      "lab-football-team-cards": {
+        "title": "Build Football Team Cards",
```

Comment:
I think this should be "Build a Set of Football Team Cards" instead of "Build Football Team Cards"

Reply:
this should be updated in all areas

Reply:
should I also update the dashed name? from `lab-football-team-cards` to -lab-set-of-football-team-cards`?

Reply:
I think the dashed name is fine to leave as is